samples/dnn/dasiamrpn_tracker.py

```diff
@@ -0,0 +1,281 @@
+            if left_pad:
+                te_im[:, 0:left_pad, :] = self.avg_chans
+            if right_pad:
+                te_im[:, c + left_pad:, :] = self.avg_chans
+            im_patch_original = te_im[int(context_ymin):int(context_ymax + 1), int(context_xmin):int(context_xmax + 1), :]
+        else:
+            im_patch_original = im[int(context_ymin):int(context_ymax + 1), int(context_xmin):int(context_xmax + 1), :]
+
+        if not np.array_equal(model_size, original_sz):
+            im_patch = cv.resize(im_patch_original, (model_size, model_size))
+        else:
+            im_patch = im_patch_original
+
+        return im_patch.astype(float)
+
+#function for reading paths, bounding box drawing, showing results
+def main():
+    parser = argparse.ArgumentParser(description="Run tracker")
+    parser.add_argument("--net", type=str, help="Full path to onnx model of net")
+    parser.add_argument("--kernel_r1", type=str, help="Full path to onnx model of kernel_r1")
+    parser.add_argument("--kernel_cls1", type=str, help="Full path to onnx model of kernel_cls1")
+    parser.add_argument("--input", type=str, help="Full path to input. Do not use if input is camera")
+    args = parser.parse_args()
+    point1 = ()
+    point2 = ()
+    mark = True
+    drawing = False
+    cx, cy, w, h = 0.0, 0.0, 0, 0
+
+    def get_bb(event, x, y, flag, param):
+        nonlocal point1, point2, cx, cy, w, h, drawing, mark
+
+        if event == cv.EVENT_LBUTTONDOWN:
+            if drawing == False:
+                drawing = True
+                point1 = (x, y)
+            else:
+                drawing = False
+
+        elif event == cv.EVENT_MOUSEMOVE:
+            if drawing == True:
+                point2 = (x, y)
+
+        elif event == cv.EVENT_LBUTTONUP:
+            cx = point1[0] - (point1[0] - point2[0]) / 2
+            cy = point1[1] - (point1[1] - point2[1]) / 2
+            w = abs(point1[0] - point2[0])
+            h = abs(point1[1] - point2[1])
+            mark = False
+
+    #loading network`s and kernel`s models
+    net = cv.dnn.readNet(args.net)
+    kernel_r1 = cv.dnn.readNet(args.kernel_r1)
+    kernel_cls1 = cv.dnn.readNet(args.kernel_cls1)
+
+    #initializing bounding box
+    cap = cv.VideoCapture(args.input if args.input else 0)
+    cv.namedWindow("DaSiamRPN")
+    cv.setMouseCallback("DaSiamRPN", get_bb)
+
+    whitespace_key = 32
+    while cv.waitKey(40) != whitespace_key:
+        ref, frame = cap.read()
+        cv.imshow("DaSiamRPN", frame)
+
+    while mark == True:
+        twin = np.copy(frame)
+        if point1 and point2:
+            cv.rectangle(twin, point1, point2, (0, 255, 255), 3)
+        cv.imshow("DaSiamRPN", twin)
+        cv.waitKey(40)
+
+    target_pos, target_sz = np.array([cx, cy]), np.array([w, h])
+    tracker = DaSiamRPNTracker(frame, target_pos, target_sz, net, kernel_r1, kernel_cls1)
+
+    #tracking loop
+    while (cap.isOpened):
```

Comment:
Please remove brackets

Reply:
It is method, not property: `cap.isOpened()`

Reply:
changed to:
`cap.isOpened()`